--- linux-6.6.110/arch/arm64/boot/dts/rockchip/Makefile.bak	2025-12-18 00:44:56.555770352 +0800
+++ linux-6.6.110/arch/arm64/boot/dts/rockchip/Makefile	2025-12-18 00:49:10.135933856 +0800
@@ -114,3 +114,4 @@
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-nanopi-r6s.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-nanopi-r6c.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3588s-rock-5a.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3568-lubancat-2n.dtb
